manifests/l/Lenovo/LeAppStore/8.1.30.526/Lenovo.LeAppStore.installer.yaml

```
@@ -7,8 +7,9 @@ MinimumOSVersion: 10.0.0.0
 InstallerType: inno
 Scope: machine
 InstallerSwitches:
-  Custom: /norestart
+  Custom: /norestart # Required, otherwise the installer would crash
```

```suggestion
  Custom: /norestart # Required, otherwise the installer would crash
  Upgrade: /norestart
```
does this also happen on upgrade?